cli/generic/forms/fields.py

```diff
@@ -250,3 +251,28 @@ def _deserialize(self, value, attr, obj, **kwargs):
             block_validated_identifiers.append(identifier)
 
         return block_validated_identifiers
+
+
+class BetField(fields.Field):
+    """
+    Implements validation of the bet.
+    """
+
+    def _deserialize(self, value, attr, obj, **kwargs):
+        """
+        Validate data (bet) that was passed to field.
+
+        Valid bet is `min` or `max` as strings, or an integer value (e.g. 15000).
+        """
+        bet = value
+
+        if bet in BET_TYPES:
+            return bet
+
+        try:
```

Comment:
Here it is not necessary, use `if - else` and `isdigit` function.
`Try - except` is slower.

Reply:
@Alladin9393, done.